modules/outbound-domain-event-handler/docker-compose.yml

```diff
@@ -0,0 +1,42 @@
+version: '3'
```

Comment:
We could have this at the top-level of the project, i.e. useful for all modules and not just the event-handler.

Otherwise we may have multiple docker-compose to do the same-thing everywhere?

Reply:
Moved to top level